src/main/scala/dpla/ingestion3/entries/ingest/ReportsEntry.scala

```diff
@@ -0,0 +1,46 @@
+package dpla.ingestion3.entries.ingest
+
+import dpla.ingestion3.confs.CmdArgs
+import dpla.ingestion3.entries.reports.ReporterMain.executeAllReports
+import dpla.ingestion3.utils.Utils
+import org.apache.spark.SparkConf
+
+/**
+  * Driver for reading DplaMapData records (mapped or enriched) and generating
+  * Reports
+  *
+  * Expects three parameters:
+  * 1) a path to the mapped/enriched data
+  * 2) a path to output the reports
+  * 3) provider short name (e.g. 'mdl', 'cdl', 'harvard')
+  *
+  * Usage
+  * -----
+  * To invoke via sbt:
+  * sbt "run-main dpla.ingestion3.ReportsEntry
+  *       --input=/input/path/to/enriched/
+  *       --output=/output/path/to/reports/
+  *       --name=shortName"
+  */
+object ReportsEntry {
+
+  def main(args: Array[String]): Unit = {
+
+    // Read in command line args.
+    val cmdArgs = new CmdArgs(args)
+
+    val dataIn = cmdArgs.getInput()
+    val dataOut = cmdArgs.getOutput()
+    val shortName = cmdArgs.getProviderName()
+
+    // Get logger
+    val logger = Utils.createLogger("reports", shortName)
+
+    val sparkConf =
+      new SparkConf()
+        .setAppName("reports")
+        .setMaster("local[*]")
```

Comment:
I think there's a way to get the Spark master URL from the config file you might want to look at, because this won't use a cluster with `local[*]`

Reply:
Good catch.  I pushed up a fix for this.  I actually copied this from the `JsonlEntry` so perhaps we should fix it there, too.